spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/validation/ValidationAutoConfiguration.java

```diff
@@ -73,6 +74,9 @@ public static MethodValidationPostProcessor methodValidationPostProcessor(Enviro
 				excludeFilters.orderedStream());
 		boolean proxyTargetClass = environment.getProperty("spring.aop.proxy-target-class", Boolean.class, true);
 		processor.setProxyTargetClass(proxyTargetClass);
+		boolean adaptConstraintViolations = environment.getProperty("spring.validation.adapt-constraint-violations",
```

Comment:
I think it's not worthy to introduce a dedicated `@ConfigurationProperties` bean for now.

Reply:
I'd prefer a `@ConfigurationProperties` bean with a single property over hand-written metadata. I also think the property should be named `spring.validation.method.adapt-constraint-violations`.

Reply:
Done.